src/sberSaluteSpeechRecognitionService.ts

```diff
@@ -35,14 +35,18 @@ export class SberSaluteSpeechRecognitionService
   private token: SberSaluteToken | null = null;
   private readonly sessionId: string;
 
-  constructor(authKey: string, sessionId?: string) {
+  constructor(
+    authKey: string,
+    sessionId?: string,
+    private readonly scope = TOKEN_SCOPE
```

Comment:
Please introduce enum with all scopes and accept scope explicitly.
Also update ReadMe file with new instructions.